src/components/tasks/card/index.tsx

```diff
@@ -538,12 +530,15 @@ const Card: FC<Props> = ({
                                         tabIndex={0}
                                     />
 
-                                    {showSuggestion && (
-                                        <SuggestionBox
-                                            suggestions={suggestions}
-                                            onClickName={handleClick}
-                                            loading={isLoadingSuggestions}
-                                        />
+                                    {isLoadingSuggestions ? (
+                                        <Loader />
+                                    ) : (
+                                        showSuggestion && (
+                                            <SuggestionBox
+                                                suggestions={suggestions}
+                                                onSelectAssignee={handleClick}
+                                            />
+                                        )
                                     )}
```

Comment:
please put it in if...else condition, it is less readable and more confusing

Reply:
@Pratiyushkumar https://stackoverflow.com/questions/44046037/if-else-statement-inside-jsx-reactjs